Modules/_math.h

```diff
@@ -23,3 +23,21 @@ _Py_log1p(double x)
 }
 
 #define m_log1p _Py_log1p
+
+static double
+_Py_fmod(double x, double y)
+{
+    /* Some platforms (e.g. Windows 10 with MSC v.1916) loose sign
+       for zero result.  But C99+ says: "if y is nonzero, the result
+       has the same sign as x".
+     */
+    double r = fmod(x, y);
+
+    if (r == 0.0 && y != 0.0) {
```

Comment:
Can you make this code specific to Windows? For example, use `#ifdef _MSC_VER`. Move the comment on this part of the code in this case.

Reply:
Do we sure that this issue specific only for Windows?